src/Styleguide/Pages/Artwork/Components/__stories__/ArtworkSidebar/ArtworkSidebarArtists.story.tsx

```diff
@@ -12,13 +12,13 @@ storiesOf("Styleguide/Artwork/Sidebar", module).add("Artists", () => {
   return (
     <React.Fragment>
       <Section title="Single Followed Artist">
-        <Artists artists={SingleFollowedArtist} />
+        <Artists artwork={{ artists: SingleFollowedArtist }} />
       </Section>
       <Section title="Single Not Followed Artist">
-        <Artists artists={SingleNonFollowedArtist} />
+        <Artists artwork={{ artists: SingleNonFollowedArtist }} />
       </Section>
       <Section title="Multipe Artists">
-        <Artists artists={MultipleArtists} />
+        <Artists artwork={{ artists: MultipleArtists }} />
```

Comment:
👍 for artwork addition here. We might need to refactor it further to support Cultural Makers on artwork, but that is a different refactor I think